docs/userguide/icu4c/faq.md

```diff
@@ -69,8 +69,8 @@ MSVC compiler. See the [Building ICU4C](./icu4c/build) page.
 
 #### Can you help me build ICU4C for ...
 
-We can try ... make sure you read the [Building ICU4C](./icu4c/build) section and also the [ICU
-Data](../icudata.md) section. You might also [searching the icu-support
```

Comment:
Several more references to icudata.md - I propose I try to update all of them in this PR